MIGRATION_GUIDE.md

```diff
@@ -3,7 +3,23 @@
 
 This guide outlines how developers can migrate from older versions of our SDK to newer ones.
 
-## Migrating from v1.X.0 to v2.0.0
+## Migrating from v2.3.0 to v2.3.1
```

Comment:
By semantic versioning, this release should be 3.0.0, as we made breaking changes to the public API.

Reply:
Yah fair maybe I can remove the old api now too.

Reply:
I'll do it in a separate PR.

Reply:
Yeah it would be a good opportunity to bundle in other things like that.